Userland/Libraries/LibVT/Terminal.cpp

```diff
@@ -1348,7 +1348,62 @@ void Terminal::handle_key_press(KeyCode key, u32 code_point, u8 flags)
         emit_string(builder.string_view());
     };
 
+    auto emit_application_code = [this](KeyCode key) {
+        // The table providing mapping from numeric keys to application keys can be found at https://vt100.net/docs/vt100-ug/chapter3.html#T3-7
+        StringView prefix = "\eO"sv;
+        StringBuilder builder;
```

Comment:
Instead of using .appendff() and eating the allocations, let's
```suggestion
        StringBuilder builder;
        builder.append("\x1bO"sv);
```
here...